docs/ibc/light-clients/localhost/integration.md

```diff
@@ -0,0 +1,32 @@
+<!--
+order: 2
+-->
+
+# Integration into Cosmos SDK chain
```

Comment:
Just to keep it consistent with other [integration sections](https://github.com/cosmos/ibc-go/blob/31b031e3c0c012705fd9fade7803aedca317ef1e/docs/apps/interchain-accounts/integration.md#integration):

```suggestion
# Integration
```